MonitorControl/AppDelegate.swift

```diff
@@ -212,12 +212,30 @@ class AppDelegate: NSObject, NSApplicationDelegate {
 
     return nil
   }
+  
+  func handleOpenPrefPane(mediaKey: MediaKey, event: KeyEvent?, modifiers: NSEvent.ModifierFlags?) -> Bool {
+    if mediaKey != .mute && mediaKey != .volumeUp && mediaKey != .volumeDown {
+      return false
+    }
+    guard let modifiers = modifiers else { return false }
+    if !modifiers.contains(.option) || modifiers.contains(.shift) {
+      return false
+    }
```

Comment:
These (except for the guard) are not really necessary, just return false by default and true after you opened the preferences (unless I'm missing something).

You can just check for the right mediakeys instead of first checking if they're not in there. Works better for sound and brightness too.